index.html

```diff
@@ -0,0 +1,60 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, intial-scale=1.0">
+   
+    
+    <link rel="stylesheet" href="./style.css" /> 
+  </head>
+    <source media="(min-width: 650px)" srcset="alps-unsplash.jpg">
+    <source media="(min-width: 650px)" srcset="img_food.jpg">
+
+  <body>
+    <header>
+  
+      <img src="./images/cropped-26090619-1-1.png"class="logo" alt="logo" > 
+  <h1>with snow in mind</h1>
+<nav>
+ <ul>
+ <li><a href="search.asp">search</a></li>
+ <li><a href="gallery">gallery</a></li>
+ <li><a href="weather.asp">weather</a></li>
+ </ul>
+</nav>
+ </header> 
+    <div class="container">
+      <h2>Top 10 ski destinations in Europe</h2>
+
+      <img src="./images/alps-unsplash.jpg" alt="alps" width="1000" height="544" />
```

Comment:
Make a <div class=""> </div> here for example with the image and the text to place them in the same box.